src/lib/fragments/AgGrid.react.js

```diff
@@ -317,6 +355,14 @@ export default class DashAgGrid extends Component {
             if (GRID_MAYBE_FUNCTIONS_NO_PARAMS[target]) {
                 return this.convertMaybeFunctionNoParams(value);
             }
+            if (target === 'alignedGrids') {
+                return (
+                    document.getElementById(value) && [
+                        findReact(document.getElementById(value)).reference
+                            .current,
```

Comment:
Isn't this ref just a DOM element? And if so, can we find it directly rather than going through React internals? Maybe it's just this?
```suggestion
                        document.getElementById(value).childNodes[0],
```

Also, the underlying API is an array, can use an array of IDs and map each one to the appropriate element? Maybe that just works for multiple aligned grids?

Reply:
No, its not mapped to the DOM, its linked on the React component. Changes to the grid makes changes to the API of the connected grid.

Yes, we could map it to an array, wasnt sure if we wanted to tackle this just yet.